src/markup_to_json.test.js

```diff
@@ -14,30 +14,13 @@ const TestCases = [
 
 const convertMarkupToJSON = require("./markup_to_json");
 
-describe("convertMarkupToJSON", () => {
-  test.each(TestCases)("converts markup to expected JSON", async (testcase) => {
-    const testMarkup = fs.readFileSync(`./test/${testcase}.markup`, "utf8");
-    const expectedJSON = JSON.parse(
-      fs.readFileSync(`./test/${testcase}.json`, "utf8")
-    );
-    const testJSON = await convertMarkupToJSON(
-      testMarkup,
-      `./test/${testcase}.source.jpg`,
-      "./literally.anywhere"
-    );
-    expect(testJSON).toEqual(expectedJSON);
-  });
-});
-
-const newConvertMarkupToJSON = require("./new_markup_to_json");
-
 describe("New Markup parser `newConvertMarkupToJSON`", () => {
```

Comment:
Oh, might want to change text here?

Reply:
newConvertMarkupToJSON to convertMarkupToJSON

Reply:
Oh good catch.

Reply:
Fixed in https://github.com/iFixit/node-markup/pull/69/commits/fe7e18fbbf47e5f271844d97c01d159413fa314e ?